prime-router/src/main/kotlin/history/db/ReportGraph.kt

```diff
@@ -133,10 +136,16 @@ class ReportGraph(
                     ITEM_LINEAGE.PARENT_REPORT_ID,
                     ITEM_LINEAGE.PARENT_INDEX,
                     DSL.value("(")
+                        .concat(ITEM_LINEAGE.CHILD_REPORT_ID)
+                        .concat(",")
+                        .concat(ITEM_LINEAGE.CHILD_INDEX)
+                        .concat(")")
+                        .concat("->(")
                         .concat(ITEM_LINEAGE.PARENT_REPORT_ID.cast(SQLDataType.VARCHAR))
                         .concat(",")
                         .concat(ITEM_LINEAGE.PARENT_INDEX)
-                        .concat(")").`as`(PATH_FIELD)
+                        .concat(")").`as`(PATH_FIELD),
+                    ITEM_LINEAGE.CHILD_REPORT_ID.`as`(STARTING_REPORT_ID_FIELD)
```

Comment:
Need to keep track of the starting report so that I can return it as part of the delivery API